card.go

```diff
@@ -33,29 +37,25 @@ type TokenizationMethod string
 // of all parameters. See AppendToAsCardSourceOrExternalAccount.
 type CardParams struct {
 	Params    `form:"*"`
-	Token     string `form:"-"`
-	Default   bool   `form:"default_for_currency"`
 	Account   string `form:"-"`
-	Customer  string `form:"-"`
-	Recipient string `form:"-"`
-	Name      string `form:"name"`
-	Number    string `form:"number"`
-	Month     string `form:"exp_month"`
-	Year      string `form:"exp_year"`
-	CVC       string `form:"-"`
-	Currency  string `form:"currency"`
 	Address1  string `form:"address_line1"`
 	Address2  string `form:"address_line2"`
+	CVC       string `form:"-"`
```

Comment:
CVC should be after Customer (I guess it's because of casing?

Reply:
Yeah. Generally with lexigraphic sorting, the capitals come first:

```
irb(main):004:0> ["Customer", "CVC"].sort
=> ["CVC", "Customer"]
```

I followed this convention here.